v2/diff_write.go

```diff
@@ -36,40 +78,63 @@ func (d DiffElement) Render(opts ...Option) string {
 		}
 	}
 	for _, oldValue := range d.Remove {
-		if isColor {
-			b.WriteString(colorRed)
+		if isVoid(oldValue) {
+			continue
 		}
-		if !isVoid(oldValue) {
+		if isSingleStringDiff && isColor {
+			oldStr := string(oldValue.(jsonString))
+			b.WriteString(" asdfasdfasd\"")
```

Comment:
:grimacing: my debug string made it into the PR -- luckily, my newly added tests discovered this bug!